xz.spec

```diff
@@ -20,7 +21,6 @@ BuildRequires: autotools
 %endif
 
 %build
-./autogen.sh
```

Comment:
This should not be dropped, and you should use the cms-externals version, as @davidlt mentioned.